Cargo.toml

```diff
@@ -28,11 +28,13 @@ futures = "0.1.11"
 futures-cpupool = "0.1"
 hyper = { git = "https://github.com/hyperium/hyper", optional = true }
 hyper-tls = { git = "https://github.com/hyperium/hyper-tls", optional = true }
+jsonwebtoken = { version = "2.0", optional = true }
 libc = "0.2.10"
 local-encoding = "0.2.0"
 log = "0.3.6"
 lru-disk-cache = { path = "lru-disk-cache" }
 number_prefix = "0.2.5"
+openssl = { version = "0.9", optional = true }
```

Comment:
Also, when removing openssl I discovered that it's actually already a dependency-- sccache depends on hyper-tls, which depends on native-tls, which depends on openssl (tokio-tls also depends on openssl).